### PR TITLE
Create LICENSE.md

### PR DESCRIPTION
**Pull request by alorma**
As I saw on this blog post, now github search in repos for License.md files, so, why not use it?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -398,21 +398,3 @@ Bugs and Feedback
 -----------------
 
 For bugs, questions and discussions please use the [Github Issues](https://github.com/SchibstedSpain/leku/issues).
-
-
-License
--------
-
-Copyright 2016 Schibsted Classified Media Spain S.L.
```
Comment (CristianGM) [inline README.md]: Why are you removing it from the readme??

Comment (CristianGM) [inline README.md]: At least add the link to the blog post to take a look

Comment (ferranpons) [inline README.md]: Mmmm yes, I am agree with that. There is no problem if it's in two places

**Comment by alorma** [comment]
Sorry, link wasn't pasted :S

https://github.com/blog/2252-license-now-displayed-on-repository-overview

**Comment by alorma** [comment]
Done @CristianGM i've removed the commit that modifies README.md

**Pull request merged**